### PR TITLE
Robusta api project

### PR DESCRIPTION
**Pull request by Sheeproid**
replacing robusta-cli python package with robusta-api python package, as robusta-cli will become it's own git repo & project at https://github.com/robusta-dev/robusta-cli.
Deleting all robusta-cli code will be included in a separate PR.

### CODE REVIEW

File: docs/_static/robusta [hunk 1]
```diff
@@ -35,8 +35,8 @@ fi
 if [[ "upgrade-cli" == "$COMMANDS" ]]; then
   docker pull us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli > /dev/null 2>&1 & spinner
   echo 'Upgrade complete'
-  docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli robusta version
+  docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli version
```
Comment (arikalon1) [inline docs/_static/robusta:38]: Is the name of the command missing here?
Do we need 
```
-v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli  robusta-cli version
```

File: docs/_static/robusta [hunk 2]
```diff
@@ -35,8 +35,8 @@ fi
...
   exit 0
 fi
 
-docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli robusta $COMMANDS
+docker run -it --rm --net host -v ~/.aws:/root/.aws -v ~/.config/gcloud:/root/.config/gcloud -v ${PWD}:/workingdir -w=/workingdir -v ~/.kube:/root/.kube us-central1-docker.pkg.dev/genuine-flight-317411/devel/robusta-cli $COMMANDS
```
Comment (arikalon1) [inline docs/_static/robusta:42]: same

File: pyproject.toml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
-name = "robusta-cli"
-version = "0.0.0"
+name = "robusta-api"
+version = "0.0.3"
```
Comment (arikalon1) [inline pyproject.toml:3]: In the example in `playbooks-repositories.rst` the version is `1.0`
what is the correct one?

**Review by arikalon1 — APPROVED** [review]
nice work
left a couple of minor comments

File: docs/_static/robusta (hunk 1 above)
Comment (Sheeproid) [reply docs/_static/robusta:38]: no, we don't need it as the new Dockerfile uses a python entrypoint

File: docs/_static/robusta (hunk 2 above)
Comment (Sheeproid) [reply docs/_static/robusta:42]: same

File: pyproject.toml (hunk 3 above)
Comment (Sheeproid) [reply pyproject.toml]: It's a mistake, it should stay 0.0.0 in the pyproject.toml file and in the example it should use ^0.14. I've fixed it.

**Pull request merged**